backend/config/database.yml

```
@@ -49,7 +49,9 @@ test:
 # for a full overview on how database connection configuration can be specified.
 #
 production:
-  <<: *default
-  database: myapp_production
-  username: myapp
-  password: <%= ENV["MYAPP_DATABASE_PASSWORD"] %>
+  database: stay_connect_db
+  adapter: postgresql
+  encoding: utf8
+  username: stay_connect_user
+  password: <%= ENV['ROOT_PASSWORD'] %>
+  host: dpg-cniid721hbls73fidp4g-a
```

環境変数名はDB_HOSTでなくても良いですが、同じ名前でrenderから環境変数を送るようにしましょう！

```suggestion
  host: <%= ENV['DB_HOST'] %>
```